.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: test hypervisor capabilities
+on:
+  push:
+    branches:
+      - ci-test-virtualization
```

Comment:
This is only here to show that [it works](https://github.com/anokfireball/gardenlinux-ccloud/actions/runs/16111166655/job/45454830223). Will remove before merging.